app/models/work_state_transition_notification.rb

```diff
@@ -15,10 +15,10 @@ def initialize(work, current_user_id)
     @work_url = Rails.application.routes.url_helpers.work_url(work)
 
     # Troubleshooting https://github.com/pulibrary/pdc_describe/issues/1783
-    if @work_url.include?("/describe/describe/")
```

Comment:
The context for this needed to be clarified for this, as https://github.com/pulibrary/pdc_describe/issues/1783 was not clear in indicating that the issue was only closed given that this *must* remain in the code base until a more optimal solution is identified.